templates/nginx.conf

```diff
@@ -60,6 +61,13 @@ server {
       proxy_set_header X-Real-IP $remote_addr;
       proxy_set_header Host $host;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+      proxy_cache lemmy_cache_{{domain|replace(".","_")|replace("-","_")}};
```

Comment:
this probably needs to ensure that at least the accept header is included in the [`proxy_cache_key`](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_key) to avoid serving `activity+json` or html to the wrong client because it's already cached